src/tools/gud.md

```diff
@@ -0,0 +1,170 @@
+# Mozilla Growth & Usage Dashboard
+#### *overall: jmccrosky@ | data: jklukas@ | frontend: hulmer@ (updated 2020-1-15)*
+___
+
+**The Mozilla Growth & Usage Dashboard is in general release and can be found [here](https://growth-stage.bespoke.nonprod.dataops.mozgcp.net/).**
+
+___
+
+# Overview
+
+The Mozilla Growth & Usage Dashboard (GUD) is a tool to visualize growth metrics in a standard way across Mozilla’s products.  It is intended to be quite intuitive to use, but we provide some brief documentation here.  Please reach out (`#gud` on slack) if anything is unclear or you are unsure how to use GUD for your purposes.
+
+# Quick Start
+
+To visualize a particular metric (such as `MAU` or `Retention`) for a particular product:
+
+1. By default, all metrics are shown.  To zoom into a particular metric, choose it from the `METRIC` selector on the left.
+
+2. Choose the product of interest from the `PRODUCT / USAGE CRITERIA` selector on the left.  Some products may have multiple possible usage criteria, corresponding to different types of use of the product.
+
+    * New profile creation is considered one of these usage criteria; thus, to determine the rate of new profile creation for a product, select that product’s “`Profile Created`” usage criteria and look at the `DAU` metric.
+
+You may also restrict to particular slices (for example, country) and to a particular date range using the additional selectors on the left.
+
+# Data Model
```

Comment:
This explanation largely overlaps with existing docs for the Exact MAU tables: https://docs.telemetry.mozilla.org/datasets/bigquery/exact_mau/reference.html

We should probably consolidate so that we don't duplicate these explanations, and make sure to reference whatever the canonical location is from the other location.

Reply:
I see your point, but also kind of feel it really needs to be in both places.  I think it's pretty essential to really understanding how to use GUD, so want to have it here, but also see that it needs to be in the exact MAU table reference.  Does markdown support an include?  Maybe that's a partial solution to at least restrict that they're in sync...  I'm also open to being convinced that one could just link to the other, but don't see it that way now.

Reply:
I agree that it's pretty central in both cases. I'm okay with duplicating for now, but perhaps we should at least make sure we point out in both articles pretty early on that a very similar data model exists for the other case, so these are obviously cross-linked. I don't want to block this on figuring out a technical solution.

Reply:
Sounds fair.  I made the change to better cross-link.  I'll wait until Friday morning to merge in case you want to review that, or any of the other reviewers want to weigh in.